src/Build/Microsoft.Build.csproj

```diff
@@ -20,7 +20,7 @@
     <IsPackable>true</IsPackable>
     <PackageDescription>This package contains the $(MSBuildProjectName) assembly which is used to create, edit, and evaluate MSBuild projects.</PackageDescription>
     <IncludeSatelliteOutputInPack>false</IncludeSatelliteOutputInPack>
-    <ApplyNgenOptimization Condition="'$(TargetFramework)' == '$(FullFrameworkTFM)'">partial</ApplyNgenOptimization>
+    <ApplyNgenOptimization Condition="'$(TargetFramework)' == '$(FullFrameworkTFM)'">full</ApplyNgenOptimization>
```

Comment:
Why are all these only on full?

I was momentarily confused as to why you'd have to change it for four projects individually rather than just one change for them all, but it doesn't really make sense to ngen anything from other assemblies, so 👍

Reply:
> Why are all these only on full?

ngen is only a thing on .NET Framework; in .NET Core it was replaced with "crossgen" (and now crossgen2) which has a different set of rules.

Reply:
I think you've told me that before...I'll try to remember this time.